app/qml/form/MMFormPage.qml

```diff
@@ -231,7 +231,7 @@ Page {
         id: editGeometry
         text: qsTr( "Edit geometry" )
         iconSource: __style.editIcon
-        visible: root.layerIsSpatial && __activeProject.projectRole !== "reader"
+        visible: root.layerIsSpatial && __activeProject.userHasEditableRole
```

Comment:
`userHasEditableRole` is now defined as `Q_INVOKABLE` method, not property. This code must be resulting in errors..